packages/web3-eth/test/integration/web3_eth/send_signed_transaction.test.ts

```diff
@@ -16,45 +16,42 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 import WebSocketProvider from 'web3-providers-ws';
-import { Address, Bytes, SignedTransactionInfoAPI, Transaction } from 'web3-types';
+import { Bytes, SignedTransactionInfoAPI, Transaction } from 'web3-types';
 import { DEFAULT_RETURN_FORMAT, FMT_BYTES, FMT_NUMBER, format, hexToNumber } from 'web3-utils';
 import { isHexStrict } from 'web3-validator';
 import { Web3Eth, InternalTransaction } from '../../../src';
-import {
-	getSystemTestAccounts,
-	getSystemTestProvider,
-	isWs,
-} from '../../fixtures/system_test_utils';
+import { createTempAccount, getSystemTestProvider, isWs } from '../../fixtures/system_test_utils';
 import { getTransactionGasPricing } from '../../../src/utils/get_transaction_gas_pricing';
 import { transactionSchema } from '../../../src/schemas';
 
 const HEX_NUMBER_DATA_FORMAT = { bytes: FMT_BYTES.HEX, number: FMT_NUMBER.HEX } as const;
 
 describe('Web3Eth.sendSignedTransaction', () => {
 	let web3Eth: Web3Eth;
-	let accounts: Address[];
+	let tempAcc: { address: string; privateKey: string };
 
-	beforeAll(async () => {
+	beforeEach(async () => {
 		web3Eth = new Web3Eth(getSystemTestProvider());
```

Comment:
It would be unnecessary to initialize `web3Eth` before each test.

I think those `beforeEach` have to be reviewed for such cases.

Reply:
fixed

Reply:
Thanks,
However, the code did not change. Did you miss pushing the code changes?
(this comment also apply to all code modifications regarding `beforeEach` and `afterEach` in all modified test cases.)

Reply:
pushed